buf/internal/lint.bzl

```diff
@@ -33,7 +33,7 @@ def _buf_lint_test_impl(ctx):
     files_to_include = []
     if ctx.file.config != None:
         files_to_include.append(ctx.file.config)
-    return protoc_plugin_test(ctx, proto_infos, ctx.executable._protoc, ctx.toolchains[_TOOLCHAIN].cli, config, files_to_include)
+    return protoc_plugin_test(ctx, proto_infos, ctx.executable._protoc, ctx.toolchains[_TOOLCHAIN].buf.protoc_lint_tool, config, files_to_include)
```

Comment:
`buf_breaking_test` and `buf_dependencies` should also be updated in a similar way.